### PR TITLE
Product details, Rating products

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by hoangminhuet** [comment]
ready

File: database/migrations/2020_07_17_063559_add_comment_column_to_ratings_table.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('ratings', function (Blueprint $table) {
+            if (!Schema::hasColumn('ratings', 'comment')) {
+                Schema::table('ratings', function (Blueprint $table) {
+                    $table->text('comment');
+                });
```
Comment (huyenttm-0901) [inline database/migrations/2020_07_17_063559_add_comment_column_to_ratings_table.php]:
```suggestion
            if (!Schema::hasColumn('ratings', 'comment')) {
                $table->text('comment');
```

**Comment by hoangminhuet** [comment]
rating

**Comment by hoangminhuet** [comment]
ready

**Pull request merged**